pages.es/linux/thunar.md

```diff
@@ -0,0 +1,16 @@
+# thunar
+
+> Administrador de archivos gráficos para entornos de escritorio XFCE.
+> Más información: <https://docs.xfce.org/xfce/thunar/start>.
+
+- Abre una nueva ventana que muestra el directorio actual:
+
+`thunar`
+
+- Abra la utilidad de cambio de nombre en bloque:
```

Comment:
I checked https://learn.microsoft.com/es-es/sql/t-sql/statements/bulk-insert-transact-sql?view=sql-server-ver16 and translates bulk to masivo
```suggestion
- Abra la utilidad de cambio de nombre masivo:
```

Reply:
I'm not sure about this alternative
```suggestion
- Abra la utilidad de cambio de nombre por lotes:
```